frontend/src/app/workspaces/[workspaceId]/workflows/[workflowId]/page.tsx

```diff
@@ -10,26 +10,10 @@ export const metadata: Metadata = {
 export default function WorkbenchPage() {
   const layout = cookies().get("react-resizable-panels:layout")
   const defaultLayout = layout ? JSON.parse(layout.value) : undefined
-  const collapsed = cookies().get("react-resizable-panels:collapsed")
```

Comment:
Are these cookies not used by the other 2 thirds of the panel?

Reply:
I don't think the other 2 thirds collapse. The collapse attribute is only relevant for the side panel?

Reply:
<img width="647" alt="Screenshot 2024-08-21 at 12 18 48 PM" src="https://github.com/user-attachments/assets/ce796172-6575-4d94-bf0a-3d3595c480c0">

Reply:
ok, i see